conf/php-fpm.ini

```diff
@@ -0,0 +1,10 @@
+; Common values to change to increase file upload limit
+; upload_max_filesize = 50M
```

Comment:
Wonder is one day, we will have to set upload_max_filesize a bit higher than default one.

Reply:
Do you think a forum need that much upload capacity?

Reply:
Mh... I don't know what is the default upload size but it's depends on the forum. Sometime user could upload Archive (tar, zip) which could have more than 50MO.

Reply:
It depends on the capabilities of the server too (enough space...).